### PR TITLE
Indian payroll

### PR DESCRIPTION
**Pull request by kusalkw**

### CODE REVIEW

File: payroll-and-expenses/sample-payroll.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+One of the best ways to decide which columns you need have in your payroll report is to think of rows in your employees payslip.
+
+Now here are the list of columns to show on the Indian payroll we are creating:
+
+1. Basic Salary
+2. Gross Salary
+3. Basic Pay
+4. province
+5. Professional Tax Slap
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: professional tax slab

File: payroll-and-expenses/sample-payroll.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+6. employee birthday
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Start with a uppercase

Comment (thilinah) [reply payroll-and-expenses/sample-payroll.md]: For other column names also

File: payroll-and-expenses/sample-payroll.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+![](../assets/in-basic-pay.png)
+
+### Adding Payroll Column for Province
+
+We need to select Get Employee Data in predefined calculations and enter province_Name in the function field in order to get employee's province
+
+![](../assets/in-province.png)
+
+### Adding Payroll Column for Professional Tax Slap
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Tax slab (not slap)

File: payroll-and-expenses/sample-payroll.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+Payslip for IceHrm Employee will look like this:
+
+
+## Generating Payroll for Next Month
+
+Once you configure your payroll for initially, generating it for the second month can be done in few minutes.
+
+### Change Dates and set status to Draft
+
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:244]: Additional space

File: payroll-and-expenses/sample-payroll.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+As shown below we have added two employees to Indian payroll
+
+
+## Configure Employee Salary Components
+
+Since the payroll depends on employee salary components you should make sure all employee salary components are defined properly. You can do this by going to **Payroll -&gt; Salary** module and selecting **Employee Salary** tab.
+
+As you can see here **IceHrm Employee** is getting a Basic and Allowance while **Lala Lamees** getting paid per hour. As we have configured our payroll report application now able to calculate the tax and other payroll columns properly.
+
+Processing Payroll Report
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: make this a title

File: payroll-and-expenses/sample-payroll.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+1. Company Logo
+2. Company Name
+3. Text  \(For adding special messages to notifications to employees\)
+4. Separators \(For separating sections on payslip\)
+5. Payroll columns
+
+
+
+## Create Payroll Report
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:191]: Payroll report is included in the payroll import, add a screenshot of this. And mention how to add new columns to payroll report if needed

Following also should be mentioned
- How to change dates of the payroll report (you cant change the dates of an existing report), so need to clone it and change dates

File: payroll-and-expenses/sample-payroll.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+Payroll Report is the unit used to combine all the payroll columns and calculate monthly payments for all the selected employees.
+
+.Goto Payroll Reports tab and create a new Payroll Report.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Why a . in front

File: payroll-and-expenses/sample-payroll.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+
+When you create the payroll report it should be in **Draft** state. Only when it is processing completed it should go to **Completed** state.
+
+Also you need to select all the payroll columns you defined earlier for Indian payroll here as shown above.
+
+## Selecting Employees For Your Payroll Report
+
+Above payroll is for monthly paid employees who are in Indian Payroll Calculation group. So you need to add some employees satisfying above requirements under **"Company Payroll"** tab.
+
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Why there are additional spaces? if its for a reason its fine

File: payroll-and-expenses/sample-payroll.md [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+
+As shown below we have added two employees to Indian payroll
+
+
+## Configure Employee Salary Components
+
+Since the payroll depends on employee salary components you should make sure all employee salary components are defined properly. You can do this by going to **Payroll -&gt; Salary** module and selecting **Employee Salary** tab.
+
+As you can see here **IceHrm Employee** is getting a Basic and Allowance while **Lala Lamees** getting paid per hour. As we have configured our payroll report application now able to calculate the tax and other payroll columns properly.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: If you are coping and pasting make sure to read the text :)

Comment (thilinah) [reply payroll-and-expenses/sample-payroll.md]: "As you can see here" ? what can you see?

File: payroll-and-expenses/sample-payroll.md [hunk 10]
```diff
@@ -0,0 +1,220 @@
...
+
+Processing Payroll Report
+
+Click on the blue color "Process" button on your payroll report under Payroll Report tab. This will show salaries of all the employees in your payroll. 
+
+As you can see here IceHrm can now calculate your payroll.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:221]: What can I see, attach a screenshot from the Indian payroll

File: payroll-and-expenses/sample-payroll.md [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+
+### Add "Net Salary" column
+
+Net Salary is Gross salary minus Total deductions. So we create Net Salary column as shown below:
+
+![](../assets/in-net-salary.png)
+
+Now we have finished defining payroll columns. When you go to Payroll columns and search "IN - " you can see all the fields in Indian payroll.
+
+## Create Payslip Template
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:173]: Create the Indian payslip with all related fields, and provide basic instructions. Then attach the Indian payroll export again to the ticket

File: payroll-and-expenses/sample-payroll.md [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+
+And then download the Payslip from any completed payroll report
+
+
+Payslip for IceHrm Employee will look like this:
+
+
+## Generating Payroll for Next Month
+
+Once you configure your payroll for initially, generating it for the second month can be done in few minutes.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:241]: Explain this process with screenshots

File: payroll-and-expenses/sample-payroll.md [hunk 13]
```diff
@@ -0,0 +1,220 @@
...
+
+### Change Dates and set status to Draft
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Change Dates and set the Status to Draft

File: payroll-and-expenses/sample-payroll.md [hunk 14]
```diff
@@ -0,0 +1,220 @@
...
+
+Click on the blue color "Process" button on your payroll report under Payroll Report tab. This will show salaries of all the employees in your payroll. 
+
+As you can see here IceHrm can now calculate your payroll.
+
+After checking figures manually you can click on finalize button which will change the payroll report status to **Completed**.
+
+## Downloading Payslips
+
+Now your employee can login and download payslip for the payroll period 2017-08-01 to 2017-08-31.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Why dates in 2017?

File: payroll-and-expenses/sample-payroll.md [hunk 15]
```diff
@@ -0,0 +1,220 @@
...
+
+Processing Payroll Report
+
+Click on the blue color "Process" button on your payroll report under Payroll Report tab. This will show salaries of all the employees in your payroll. 
+
+As you can see here IceHrm can now calculate your payroll.
+
+After checking figures manually you can click on finalize button which will change the payroll report status to **Completed**.
+
+## Downloading Payslips
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:227]: Explain with screenshots and attach a screenshot of a generated payslip

File: payroll-and-expenses/sample-payroll.md [hunk 16]
```diff
@@ -0,0 +1,220 @@
...
+### Adding Payroll Column for Gross Salary
+
+![](../assets/in-gross-salary.png)
+
+### Adding Payroll Column for Basic Pay
+
+Based on the previous column you have added, now you can create the hourly pay based column as shown below:
+
+First create a new payroll column.Then, click on "Add" button next to Calculation Columns
+. Here you are adding the previously defined **gross salary column as a parameter named X**  so you can use it to do various calculations using these columns
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: Why starting with a .

File: payroll-and-expenses/sample-payroll.md [hunk 17]
```diff
@@ -0,0 +1,220 @@
...
+![](../assets/in-income-tax.png)
+
+### Adding Payroll Column for ESI employee contribution
+
+
+As you have seen you can do some calculations at column level. But things such as tax which are having different percentages and multiple slabs it's better to use saved calculations. Now you should go to **Saved Calculations** tab and add a new saved calculation.
+
+You can create a saved calculation based on a Salary component group \(type\), a Salary component or an Existing payroll column. In this case we use an existing payroll column.
+
+Now click on add button on Calculation Process filed to define the actual calculation. According to our example for the full range of esi is 0.75%. So we don't need to define any ranges and can calculate the tax as follows.
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md]: esi => ESI

File: payroll-and-expenses/sample-payroll.md [hunk 18]
```diff
@@ -0,0 +1,220 @@
+# Create a sample Payroll
```
Comment (thilinah) [inline payroll-and-expenses/sample-payroll.md:1]: Where did you add  the information about having to add a custom field to the employee

**Review by thilinah — CHANGES_REQUESTED** [review]
Changes needed

File: payroll-and-expenses/sample-payroll.md (hunk 6 above)
Comment (kusalkw) [reply payroll-and-expenses/sample-payroll.md:191]: can't we change the dates of an existing report?? It can be done

**Pull request merged**